src/main/java/com/fasterxml/jackson/core/json/ByteSourceJsonBootstrapper.java

```diff
@@ -252,7 +254,7 @@ public JsonParser constructParser(ObjectReadContext readCtxt,
                 ByteQuadsCanonicalizer can = rootByteSymbols.makeChild(factoryFeatures);
                 return new UTF8StreamJsonParser(readCtxt, _context,
                         streamReadFeatures, formatReadFeatures, _in, can,
-                        _inputBuffer, _inputPtr, _inputEnd, _bufferRecyclable);
+                        _inputBuffer, _inputPtr, _inputEnd, bytesProcessed, _bufferRecyclable);
```

Comment:
Hmmh. This is bit problematic wrt backwards-compatibility. Need to think about it, needs to go in 2.10.0 not 2.9.x.

Reply:
Problematic because `UTF8StreamJsonParser` no longer has a constructor with 9 arguments (it has 10 now)? I could add a `UTF8StreamJsonParser` public constructor that defaults `bytesProcessed` to 0.

Reply:
Right, only for that reason. So although technically one is supposed to always make sure jackson-core / jackson-databind minor versions align, we try to make sure that "adjacent" versions still have implementations of deprecated methods for at least one version.

So in this case keeping deprecated "old" version around would allow 2.9/2.10 combination to still work (2.9 databind, 2.10 core).

Reply:
Going to add back 10 arg constructors.
Is it really worth deprecating that 10 args constructor in favor of the 11 args one?

Reply:
10 arg constructor is back!